specs/error_state/error_write_protection.md

```diff
@@ -0,0 +1,25 @@
+# ErrorWriteProtection state
+
+## Procedure
+there are some op codes which modify state. there are `[SSTORE, CREATE, CREATE2, 
+CALL, SELFDESTRUCT, LOG0, LOG1, LOG2, LOG3, LOG4]`
+when execution call context is ready only (static call), these op codes running will encounter
+write protection error.
+### EVM behavior
+in above op codes which modify state, `CALL` is somewhat special. for non call op codes(SSTORE, CREATE, etc)
+first check if running in ready only call context, if yes, throw write protection error.
+for `CALL` op code, it will also check `value` is not zero. only both non zero value & ready only call context, then
```

Comment:
ready only => read only

Reply:
fixed in [7233568](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/365/commits/7233568918063a0170ea894d7e7219e9e822fce6) thanks.